apps/src/TrimRange.C

```diff
@@ -10,6 +10,13 @@ TrimRange::TrimRange(SimconfType * simconf, SampleBase * sample) :
 }
 
 void
+TrimRange::vacancyCreation()
+{
+  // Quick calculation of Damage
+  _simconf->vacancies_created += _recoil->_E * 0.4 / _element->_Edisp;
```

Comment:
we should actually do this
http://www.iue.tuwien.ac.at/phd/hoessinger/node47.html